common/datavalues/src/types/deserializations/nullable.rs

```diff
@@ -84,47 +84,66 @@ impl TypeDeserializer for NullableDeserializer {
     }
 
     // TODO: support null text setting
-    fn de_text<R: BufferRead>(&mut self, reader: &mut R, format: &FormatSettings) -> Result<()> {
+    fn de_text<R: BufferRead>(
+        &mut self,
+        reader: &mut NestedCheckpointReader<R>,
+        format: &FormatSettings,
+    ) -> Result<()> {
         if reader.ignore_insensitive_bytes(b"null")? {
+            reader.pop_checkpoint();
```

Comment:
No `push` before this code, why pop here?

Reply:
my fault, seems tests does not cover here

Reply:
fixed